### PR TITLE
Feature added scrolltotop

### LINKED ISSUE #2258: [handbook] Add a back to the top button
**Current Behavior**
There is no back to the top button existing on the community handbook. 

**Desired Situation**
Add a back to the top button on the handbook to help users scroll through the page.
A typical example can be found on [this site](https://smp-spec.io/).

---
**Screenshot**

Screenshot 2021-10-12 at 02 06 19

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Sayantan-s**
**Description**
- Added Scrolltotop-button component in the components folder.
- Rendered the component in the layout component.
- Conditional rendering of the button on scrolling.

Signed-off-by: Sayantan Samanta [sssmanta789@gmail.com](sssmanta789@gmail.com)

This PR fixes #2258

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,5 +77,6 @@ yarn.lock
 
 #IDE Files
 .vscode
+access.txt
```
Comment (leecalcote) [inline .gitignore:80]: @Sayantan-s, this is probably unnecessary. Will you take a moment to remove this?

File: gatsby-browser.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 import { wrapRootElement as wrap} from "./root-wrapper";
 import "./fonts.css";
 
-export const wrapRootElement = wrap;
+export const wrapRootElement = wrap;
```
Comment (leecalcote) [inline gatsby-browser.js:4]: @Sayantan-s let's avoid any changes here. Please revert this change.

**Review by leecalcote — COMMENTED** [review]
Thanks for jumping in here, @Sayantan-s 👍

File: .gitignore (hunk 1 above)
Comment (Sayantan-s) [reply .gitignore:80]: @leecalcote yeah, I'll remove this rn

**Comment by l5io** [comment]
🚀 Preview for commit 4f9ab0567fb64ae71c4ad5dd1ed1a245cc567b3e at: https://618ea9dd2779ff0c69382dbb--layer5ng.netlify.app

File: src/components/Scrolltotop-button/ScrollToTopStyles.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    right : 1.5rem;
+    outline : none;
+    border-radius:  50%;
+    border : none;
+    animation-name : ${props => props.isMounted ? onMountBtnAnimation : onUnmountBtnAnimation};
+    animation-duration: ${props => `${props.delay}ms`};
+    background-color: #3c494f;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.26);
+    cursor: pointer;
+`;
```
Comment (Nikhil-Ladha) [inline src/components/Scrolltotop-button/ScrollToTopStyles.js:17]: You need to increase the z-index for the button, it hides under other some sections across various pages.

Comment (Sayantan-s) [reply src/components/Scrolltotop-button/ScrollToTopStyles.js:18]: @Nikhil-Ladha done.

**Comment by l5io** [comment]
🚀 Preview for commit eddfa9aa2f4cd72115f7876fc8364efe2b2ebc19 at: https://6192ad7f1590e41ac026ee2b--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉 Thank you @Sayantan-s

File: gatsby-browser.js (hunk 2 above)
Comment (leecalcote) [reply gatsby-browser.js:4]: @Sayantan-s, send word when this update has been made.

**Comment by leecalcote** [comment]
Good work, @Sayantan-s 👍
